gistUpload.js

```diff
@@ -1,7 +1,8 @@
 const axios = require('axios');
-const fs = require('fs');
+const fileSystem = require('fs');
 
 const API_URL = 'https://api.github.com/gists';
+let finalString = '';
```

Comment:
What will occur if you call `readFile('someFile', 'someFile')`, and then `readFile('someOtherFile', 'someOtherFile')` ?

Reply:
The same file gets uploaded twice, once with the different gist names.